tests/integration/test_package_managers.py

```diff
@@ -52,6 +52,32 @@
             ),
             id="gomod_vendored_with_flag",
         ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/gomod-vendor-check-no-vendor.git",
+                ref="7ba383d5592910edbf7f287d4b5a00c5ababf751",
+                packages=({"path": ".", "type": "gomod"},),
+                check_output_json=True,
+                check_deps_checksums=True,
+                vendor_check_flag=True,
+                expected_rc=0,
+                expected_output="All dependencies fetched successfully",
+            ),
+            id="gomod_vendor_check_no_vendor",
+        ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/gomod-vendor-check-pass.git",
+                ref="0543a5034b687df174c6b12b7b6b9c04770a856f",
+                packages=({"path": ".", "type": "gomod"},),
+                check_output_json=True,
+                check_deps_checksums=True,
+                vendor_check_flag=True,
```

Comment:
Maybe specifying a list of flags would be easier?

```python
flags=["--gomod-vendor-check"]
```

Reply:
ack